README.md

```diff
@@ -27,6 +27,23 @@ centrally managing routing policy and fronting service discovery.
 * [linkerd](linkerd/README.md)
 * [namerd](namerd/README.md)
 
+## Quick start ##
+
+### Boot linkerd ###
+
+```
+$ ./sbt linkerd-examples/http:run
+$ open http://localhost:9990
+```
+
+### Boot namerd ###
+
+```
+$ ./sbt namerd-examples/basic:run
+$ open http://localhost:9991
+$ curl :4180/api/1/dtabs
+```
```

Comment:
Is this helpful? If this is for the uninitiated, there's no explanation of what's going on.  If this is for the initiated... is this helpful?